project/src/analysis/rdf.py

```diff
@@ -0,0 +1,35 @@
+"""Facilitate the calculation of the RDF from simulation data."""
+
+import freud
+import gsd
+import gsd.hoomd
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def gsd_rdf(job):
+    """Compute the RDF given a GSD file."""
+    gsdfile = job.fn("trajectory.gsd")
+
+    # TODO check that these are reasonable values
```

Comment:
Might want to make these function arguments with these as the default values?

Reply:
just to make sure I understand--do you mean something like
```python
def gsd_rdf(job, fname="trajectory.gsd"):
    """Compute the RDF given a GSD file."""
    gsdfile = job.fn(fname)
```
?

Reply:
On further thought, I think you mean
```
def gsd_rdf(job, frames = 10, bins = 50, r_min = 0.5, r_max = 2.5):
```
and I agree!

Reply:
Yeah this is what I meant!